README.md

```diff
@@ -85,6 +85,79 @@ sqlAPI.runSQL("<tiledb-cloud-username>", sqlDetails).then((res) => {
 
 ```
 
+
+## Cap'n Proto serialization
+
+Endpoints that support cap'n proto mime type user should set content-type to `application/capnp`.
+
+For `POST` requests library will automatically serialize data to cap'n proto.
+
+```
+const tiledb = require("@tiledb-inc/tiledb-cloud");
+
+const config = new tiledb.Configuration({
+  apiKey: "<insert token from setting page here>"
+});
+const arrayApi = new tiledb.ArrayApi(config);
+
+arrayApi
+  .updateArrayMetadataCap(
+    "ns",
+    "array_name",
+    data,
+    {
+      headers: {
+        "Content-Type": "application/capnp",
+      },
+    })
+```
+
+For `GET` requests library provides methods to deserialize data. In that case, **responseType** should be set to `arraybuffer` since helpers accept [ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer) to deserialize the response.
+
+```
+const tiledb = require("@tiledb-inc/tiledb-cloud");
+
+const config = new tiledb.Configuration({
+  apiKey: "<insert token from setting page here>"
+});
+const arrayApi = new tiledb.ArrayApi(config);
+
+arrayApi
+  .getArrayMetadataCap("ns", "array_name", {
+    headers: {
+      "Content-Type": "application/capnp",
+    },
+    responseType: "arraybuffer",
+  })
+  .then((data) => {
+    console.log(tiledb.capnpArrayMetadaDeSerializer(data.data));
+  })
+```
+
+## Development
+
+For capnproto encoding you need to [install capnproto](https://capnproto.org/install.html)
+
+Debian / Ubuntu: `apt-get install capnproto` <br/>
+Arch Linux: `sudo pacman -S capnproto` <br/>
+Homebrew (OSX): `brew install capnp`
+
+and the schema compiler
+
+```
+npm install -g capnpc-ts
+```
+
+Then run:
+```
+npm run capnp
+```
+
+
+## Known issues
+
+In the browser environment, `application/capnp` doesn't work for `GET` requests, since [Axios deletes content-type header](https://github.com/axios/axios/blob/master/lib/adapters/xhr.js#L140)
```

Comment:
Would `Accept` header work here?

Reply:
I think it should, i just tried it and i can see the `Accept` header.